README.md

```diff
@@ -31,82 +69,134 @@ Add this plugin like this:
 
 ## Example usage
 
-Let's start by creating a **root meta feed**, necessary for using this module.
-It lives alongside your existing "classical" feed, which we'll refer to as
-**main feed**.
-
+We create a subfeed for messages which describe us under our `root` feed
 ```js
-sbot.metafeeds.findOrCreate((err, metafeed) => {
-  console.log(metafeed) // { seed, keys }
+const find = (feed) => (
+  feed.feedpurpose === 'aboutMe' &&
+  feed.feedformat === 'classic'
+)
+const details = {
+  feedpurpose: 'aboutMe',
+  feedformat: 'classic',
+}
+
+sbot.metafeeds.findOrCreate(null, find, details, (err, aboutMeFeed) => {
+  console.log(rootFeed)
+
+  // publish things to aboutMeFeed!
```

Comment:
Gave the simplest most common use example I could think of.
Showed how to use the feed object thingy to publish something